common/components/chrome/SubHeader.jsx

```diff
@@ -262,23 +262,31 @@ class SubHeader extends React.Component<{||}, State > {
   }
 
   _renderHeaderLinks(): React$Node {
-    return FooterLinks.list().map((link, i) =>
-      <div className="SectionLink-root" key={i}>
-        <a className="SubHeader-anchor" href={link.url} target="_blank" rel="noopener noreferrer">
-          <h3>{link.name}</h3>
-        </a>
-      </div>
+    //TODO: Extract the links into a component much like FooterLinks
```

Comment:
The FooterLinks component had a Donate button added to it, so we were getting two donate buttons in the header. In the future I'd like to see this refactored to be not hardcoded links but since we only have two, I did it for the sake of releasing this week. Your code was perfectly solid, but the data it drew from changed in another PR.